pkg/versionconstants/versionconstants.go

```diff
@@ -11,7 +11,7 @@ var AmplitudeAPIKey = ""
 var WebImg = "ddev/ddev-webserver"
 
 // WebTag defines the default web image tag
-var WebTag = "20241002_deviantintegral_vim_tiny" // Note that this can be overridden by make
+var WebTag = "20240927_andriokha_nginx_headers" // Note that this can be overridden by make
```

Comment:
Sorry @andriokha this was a mistake to have required this. Now this will get in easier :)

```suggestion
var WebTag = "20241002_deviantintegral_vim_tiny" // Note that this can be overridden by make
```